packages/angular-cli/commands/test.ts

```diff
@@ -5,6 +5,7 @@ import {CliConfig} from '../models/config';
 const NgCliTestCommand = TestCommand.extend({
   availableOptions: [
     { name: 'watch', type: Boolean, default: true, aliases: ['w'] },
+    { name: 'debug', type: Boolean, default: false, aliases: ['d'] },
```

Comment:
i actually implemented debug false as default so it doesn't need to be passed. i added an argument to the test command --debug (-d). I think we would add a npm script npm run test:debug which would  call ng test --debug